ui/pages/routes/routes.component.tsx

```diff
@@ -656,19 +656,61 @@ export default function Routes() {
               );
             }}
           </Route>
-          <Authenticated
-            path={`${ASSET_ROUTE}/image/:asset/:id`}
-            component={NftFullImage}
-          />
-          <Authenticated
-            path={`${ASSET_ROUTE}/:chainId/:asset/:id`}
-            component={Asset}
-          />
-          <Authenticated
-            path={`${ASSET_ROUTE}/:chainId/:asset/`}
-            component={Asset}
-          />
-          <Authenticated path={`${ASSET_ROUTE}/:chainId`} component={Asset} />
+          <Route path={`${ASSET_ROUTE}/image/:asset/:id`}>
```

Comment:
Routes extract params from match object and pass them as props since v5-compat `useParams()` doesn't work inside v5 Route contexts